lib/rubocop/cop/security/json_load.rb

```diff
@@ -0,0 +1,84 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Security
+      # This cop checks for the use of unsecure JSON methods.
+      #
+      # @example
+      #   # always offense
+      #   JSON.load("{}")
+      #   JSON.restore("{}")
+      #
+      #   # no offense
+      #   JSON.parse("{}")
+      #
+      class JSONLoad < Cop
+        DANGEROUS_METHODS = [:load, :restore].freeze
+
+        MSG = 'Do not use `JSON#%s`. Use `JSON#parse instead`.'.freeze
```

Comment:
You've got a misplaced backtick near `JSON#parse`. As this is a class method it should actually be `JSON.parse`, btw.

You can also shorten the sentence to "Use X instead of Y.".